lib/treatments.js

```diff
@@ -88,9 +123,9 @@ function storage (env, ctx) {
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1874462250/issues/source?bid=2452374&fileBranchId=3087783#l124)